src/storage/kv/transaction.rs

```diff
@@ -732,20 +729,20 @@ impl Transaction {
             }
 
             // Create a new value reference and decode the value.
-            let mut val_ref = ValueRef::new(self.core.clone());
+            // let mut val_ref = ValueRef::new(self.core.clone());
```

Comment:
Leftover? Probably this entire section needs to be removed?

Reply:
Removed.